syntax_and_semantics/operators.md

```diff
@@ -21,71 +38,201 @@ struct Vector2
-* `|` – bitwise or
-* `^` – bitwise xor
-* `**` – exponentiation
-* `<<` – shift left, append
-* `>>` – shift right
-* `==` – equals
-* `!=` – not equals
-* `<` – less
-* `<=` – less or equal
-* `>` – greater
-* `>=` – greater or equal
-* `<=>` – comparison
-* `===` – [case equality](case.html)
+The conditional operator (`? :`) however, is technically a ternary operator.
+But it is not implemented as such, the compiler simply transforms it to an `if`
+condition.
 
-## Indexing
+## List of Operators
 
-```crystal
-[]  # array index (raises on out of bounds)
-[]? # array index (nil on out of bounds)
-[]= # array index assignment
-```
+This list is sorted by precedence, upper entries bind stronger than lower ones.
+
+| Category        | Operators                                            |
+|-----------------|------------------------------------------------------|
+| Index accessors | `[]`, `[]?`                                          |
+| Unary           | `+`, `&+`, `-`, `&-`, `!`, `~`, `*`, `**`, `&`, `->` |
+| Exponential     | `**`, `&**`                                          |
+| Multiplicative  | `*`, `&*`, `/`, `//`, `%`                            |
+| Additive        | `+`, `&+`, `-`, `&-`                                 |
+| Shift           | `<<`, `>>`                                           |
+| Binary AND      | `&`                                                  |
+| Binary OR/XOR   | `\|`, `^`                                            |
+| Equality        | `==`, `!=`, `=~`, `!~`, `===`                        |
+| Comparison      | `<`, `<=`, `>`, `>=`, `<=>`                          |
+| Logical AND     | `&&`                                                 |
+| Logical OR      | `\|\|`                                               |
+| Range           | `..`, `...`                                          |
+| Conditional     | `?:`                                                 |
+| Assignment      | `=`, `[]=`, `+=`, `&+=`, `-=`, `&-=`, `*=`, `&*=`, `/=`, `//=`, `%=`, `\|=`, `&=`, `^=`, `**=`, `<<=`, `>>=`, `\|\|=`, `&&=` |
+
+## Operators explained
+### Arithmetic operators
+#### Unary
+
+| Operator | Description | Example | Overloadable |
+|---|---|---|---|
+| `+`  | positive | `+1` | yes |
+| `&+` | positive (overflowing) | `&+1` | yes |
+| `-`  | negative | `-1` | yes |
+| `&-` | negative (overflowing) | `&-1` | yes |
+
+#### Multiplicative
+| Operator | Description | Example | Overloadable |
+|---|---|---|---|
+| `**` | exponentiation | `1 ** 2` | yes |
+| `&**` | exponentiation (overflowing) | `1 &** 2` | yes |
+| `*` | multiplication | `1 * 2` | yes |
+| `&*` | multiplication (overflowing) | `1 &* 2` | yes |
+| `/` | division | `1 / 2` | yes |
+| `//` | floor division | `1 // 2` | yes |
+| `%` | modulus | `1 % 2` | yes |
+
+#### Additive
+| Operator | Description | Example | Overloadable |
+|---|---|---|---|
+| `+` | addition | `1 + 2` | yes |
+| `&+` | addition (overflowing) | `1 &+ 2` | yes |
```

Comment:
I think is either `Overflow addition` (swift uses that term) or `wrapping addition`.  It's weird to me that the overflowing addition does not raise overflow (but it might be my Spanish getting in the middle).